dotnet-7.yaml

```diff
@@ -23,7 +23,6 @@ environment:
       - busybox
       - ca-certificates-bundle
       - clang-15
-      - clang-15-default
```

Comment:
I think this one might have been missed. We'd need an epoch bump here + the renames, since the llvm packages were left untouched.

Reply:
Urgh - this was hangover from me splitting out dotnet-7 as it FTBFS